dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/audit/TrackedEntityAuditService.java

```diff
@@ -40,11 +41,11 @@ public interface TrackedEntityAuditService {
 
   String ID = TrackedEntityAuditService.class.getName();
 
-  void addTrackedEntityAudit(
-      TrackedEntity trackedEntity, String username, AuditOperationType auditOperationType);
+  void addTrackedEntityAudit(AuditOperationType type, String username, TrackedEntity trackedEntity);
```

Comment:
trackedEntity here could be `@Nonnull` as well?

Reply:
TE does not need to be non-null see implementation